src/electionguard/logs.py

```diff
@@ -44,6 +49,13 @@ def __formatted_message(self, message: str) -> str:
         message = f"{os.path.basename(filename)}.{funcname}:#L{line}: {message}"
         return message
 
+    def set_stream_log_level(self, Level: int) -> None:
```

Comment:
Previously the stream log level was hard-coded to INFO.  It is now settable since GUI will log at DEBUG.